optaweb-employee-rostering-frontend/src/ui/Alerts.test.tsx

```diff
@@ -85,7 +85,7 @@ const someAlerts: Props = {
       id: 0,
       createdAt: date,
       i18nKey: 'infoMessage',
-      variant: 'info',
+      variant: 'info' as 'info',
```

Comment:
This seems redundant. I confirm that I saw an error after removing this at some point but now it works without the assertion (on my machine). Can you check and remove the assertion if it works for you as well?